app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Comment:
- [ ] It would be better if the `foreign_key: :author_id` was removed as mentioned in the previous review :man_technologist: